packages/rn-vector/package.json

```diff
@@ -63,5 +63,11 @@
         "react-native": "0.70.0",
         "react-native-vector-icons": "^9.2.0",
         "typescript": "^4.8.3"
-    }
+    },
+    "dependencies": {
+        "@brightlayer-ui/prettier-config": "^1.0.3",
+        "eslint-config-prettier": "^8.5.0",
+        "prettier": "^2.8.1"
```

Comment:
these need to be in devDependencies only.

'dependencies' implies that we need to install these packages for our icons to work at runtime, but we only use these to format the code prior to building and publishing.